osu.Game/Audio/PreviewTrack.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Audio.Track;
+
+namespace osu.Game.Audio
+{
+    public class PreviewTrack
```

Comment:
This should probably be a component too.

Reply:
Any reason this doesn't just derive from `Track` rather than own a `Track`?

Reply:
Don't know the easy way to get `PreviewTrack : TrackBass` object from [here](https://github.com/ppy/osu-framework/blob/master/osu.Framework/Audio/Track/TrackManager.cs#L17)